css/style.css

```diff
@@ -1,15 +1,15 @@
 @import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:700,300,300italic|Roboto+Mono);
 * {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
 }
 
 body {
-  font-family: 'Lato', sans-serif;
-  font-size: 18px;
-  line-height: 1.45;
-  background: #48A2DF;
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    line-height: 1.45;
+    background: #48A2DF;
```

Comment:
I thought the project was moving to two space indentation.  Maybe this was all done automatically by the tool you showed me yesterday?